src/components/Card/Card.stories.tsx

```diff
@@ -1,135 +1,41 @@
 import * as React from "react";
-import { action } from "@storybook/addon-actions";
-import { text, boolean, select } from "@storybook/addon-knobs";
 
-import Card from "./Card";
-import Heading from "../Heading/Heading";
-import Image from "../Image/Image";
-import Button from "../Button/Button";
+import Card, { CardImage, CardHeading, CardContent, CardActions } from "./Card";
+
 import Link from "../Link/Link";
-import Icon from "../Icons/Icon";
-import { IconRotationTypes, IconNames } from "../Icons/IconTypes";
 import { LinkTypes } from "../Link/LinkTypes";
-import { ButtonTypes } from "../Button/ButtonTypes";
 import { getCategory } from "../../utils/componentCategories";
 
 export default {
   title: getCategory("Card"),
   component: Card,
 };
 
-let showCTAs, showImage, showHeader, showFooter;
-
-const headerLevels = [1, 2, 3, 4, 5, 6];
-
-const imageRatios = {
-  "1x1": "https://placeimg.com/100/100/animals",
-  "2x1": "https://placeimg.com/200/100/animals",
-  "3x4": "https://placeimg.com/150/200/animals",
-  "4x1": "https://placeimg.com/400/100/animals",
-  "4x3": "https://placeimg.com/200/150/animals",
-  "16x9": "https://placeimg.com/400/225/animals",
-};
-
-export const card = () => (
-  <>
-    {boolean("Show Header", true) ? (showHeader = true) : (showHeader = false)}
-    {boolean("Show Image", true) ? (showImage = true) : (showImage = false)}
-    {boolean("Show CTAs", true) ? (showCTAs = true) : (showCTAs = false)}
-    {boolean("Show Footer", true) ? (showFooter = true) : (showFooter = false)}
-    <Card
-      id="cardID"
-      heading={
-        showHeader ? (
-          <Heading
-            level={select("Header Level", headerLevels, headerLevels[2])}
-            id="heading1"
-            text={text("Header Content", "Optional Header")}
-          />
-        ) : (
-          false
-        )
-      }
-      image={
-        showImage ? (
-          <Image
-            src={select("Image Ratio", imageRatios, imageRatios[2])}
-            alt={""}
-          />
-        ) : null
-      }
-      ctas={
-        showCTAs ? (
-          <Button
-            onClick={action("clicked")}
-            id="button1"
-            buttonType={ButtonTypes.Primary}
-            type="submit"
-          >
-            {text("CTA Button Text: ", "Example CTA")}
-          </Button>
-        ) : null
-      }
-      footer={
-        showFooter ? <>{text("Footer content: ", "Optional footer")}</> : null
-      }
-    >
-      {text("Card Content: ", "Middle column content")}
-    </Card>
-  </>
-);
-
-card.storyName = "Card";
-
-card.parameters = {
-  design: {
-    type: "figma",
-    url:
-      "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=17167%3A58131",
-  },
-};
-
 export const ExampleCard = () => (
```

Comment:
This should be moved to the mdx file and this file can be deleted.

Reply:
I did not move this code because the `ExampleCard` component was not really being used in the stories of other components.  However, I did removed the `.tsx` story file altogether.